### ISSUE TITLE
"Automatically check for updates" & "Update channel" dropdowns are appearing with green color border in High contrast black and brown color border in High contrast white.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#353]
### Steps to Reproduce

Pre-Requisite: Change the system high contrast mode before launching the application.

1. Launch Xamarin Workbooks application.
2. Navigate to Tools and Preferences. Invoke preferences tab. It will open the general tab of preferences. Invoke the update tab item using mouse (As tab items are not accessible by keyboard)
3. Verify the border color of "Automatically check for updates" & update channel dropdowns in both modes of high contrast
### Expected Behavior
Border color should follow high contrast standards
### Actual Behavior
"Automatically check for updates" & "update channel" dropdowns border is appearing in green color in High contrast black and brown color in high contrast white, but it is not a disabled button. User may perceive it as disabled button because of green color border.
### User Impact
As the border color appears in green color in HC Black and brown in HC White, user may perceive it as disabled element which is actually not. It deviates the user from the purpose.
### Screenshots
![11 hc black](https://user-images.githubusercontent.com/22956130/40723736-2cbce758-643d-11e8-8e0d-b6c4ce3c1107.PNG)
![11 hc white](https://user-images.githubusercontent.com/22956130/40723738-2cea8032-643d-11e8-9051-d110dd08a16e.PNG)
![11 normal mode](https://user-images.githubusercontent.com/22956130/40723740-2d159240-643d-11e8-884a-8e08340c0fea.PNG)

### Environment
Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
